### PR TITLE
Alerting eds design

### PR DESCRIPTION
**Pull request by nilsgstrabo**

### CODE REVIEW

File: src/components/alerting/style.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  display: grid;
+  grid-template-columns: 1fr auto;
+  gap: var(--eds_spacing_small);
+}
+
+.alerting-status {
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-gap: var(--eds_spacing_small);
+  align-items: start;
```
Comment (oleterno-equinor) [inline src/components/alerting/style.css:11]:
```suggestion
  align-items: center;
```

File: src/components/page-environment/environment-alerting.js [hunk 2]
```diff
@@ -91,24 +91,24 @@ const EnvironmentAlerting = ({
...
             <Typography as="span" bold>
               {alertingConfig?.enabled ? 'enabled' : 'disabled'}
-            </Typography>
-          </Typography>
-          <Typography>
-            <Button variant="outlined" onClick={() => setVisibleScrim(true)}>
-              Configure
+            </Typography>{' '}
+            <Button variant="ghost" onClick={() => setVisibleScrim(true)}>
+              Setup alert
```
Comment (oleterno-equinor) [inline src/components/page-environment/environment-alerting.js:99]: When alerting is already setup, say something like `Edit alert`?
```suggestion
              {alertingConfig?.enabled ? 'Edit alert' : 'Setup alert'}
```

File: src/components/page-pipeline-jobs/application-alerting.js [hunk 3]
```diff
@@ -86,26 +86,24 @@ const ApplicationAlerting = ({
...
-            </Typography>
-            <Button
-              style={{ marginLeft: '16px' }}
-              variant="outlined"
-              onClick={() => setVisibleScrim(true)}
-            >
-              Configure
+            </Typography>{' '}
+            <Button variant="ghost" onClick={() => setVisibleScrim(true)}>
+              Setup alert
```
Comment (oleterno-equinor) [inline src/components/page-pipeline-jobs/application-alerting.js:94]: When alerting is already setup, say something like `Edit alert`?
```suggestion
              {alertingConfig?.enabled ? 'Edit alert' : 'Setup alert'}
```

File: src/components/page-environment/environment-alerting.js (hunk 2 above)
Comment (nilsgstrabo) [reply src/components/page-environment/environment-alerting.js:99]: Good suggestion. Fixed

File: src/components/page-pipeline-jobs/application-alerting.js (hunk 3 above)
Comment (nilsgstrabo) [reply src/components/page-pipeline-jobs/application-alerting.js:94]: Fixed

File: src/components/alerting/style.css (hunk 1 above)
Comment (nilsgstrabo) [reply src/components/alerting/style.css]: fixed

**Review by oleterno-equinor — APPROVED** [review]

**Pull request merged**